tests/CloseConflictingAppsTests.js

```diff
@@ -112,38 +107,43 @@ gpii.tests.conflictingApps.testDefs = [
-                    },
-
+                    }
+                ]
+            },
+            "gpii.settingsHandlers.JAWSSettingsHandler": {
+                "com.freedomscientific.jaws": [
                     {
                         "settings": {
-                            "ENU-Global.Rate": 400,
-                            "ENU-Global.Punctuation": 3,
-                            "ENU-Global.Pitch": 11,
-                            "ENU-Message.Rate": 400,
-                            "ENU-Message.Punctuation": 3,
-                            "ENU-Message.Pitch": 11,
-                            "ENU-Keyboard.Rate": 400,
-                            "ENU-Keyboard.Punctuation": 3,
-                            "ENU-Keyboard.Pitch": 11,
-                            "ENU-PCCursor.Rate": 400,
-                            "ENU-PCCursor.Punctuation": 3,
-                            "ENU-PCCursor.Pitch": 11,
-                            "ENU-JAWSCursor.Rate": 400,
-                            "ENU-JAWSCursor.Punctuation": 3,
-                            "ENU-JAWSCursor.Pitch": 11,
-                            "ENU-MenuAndDialog.Rate": 400,
-                            "ENU-MenuAndDialog.Punctuation": 3,
-                            "ENU-MenuAndDialog.Pitch": 11
+                            // NOTE: Commented until GPII-4336 is addressed
+                            // ============================================
+                            // "ENU-Global.Rate": 400,
+                            // "ENU-Global.Punctuation": 3,
+                            // "ENU-Global.Pitch": 11,
+                            // "ENU-Message.Rate": 400,
+                            // "ENU-Message.Punctuation": 3,
+                            // "ENU-Message.Pitch": 11,
+                            // "ENU-Keyboard.Rate": 400,
+                            // "ENU-Keyboard.Punctuation": 3,
+                            // "ENU-Keyboard.Pitch": 11,
+                            // "ENU-PCCursor.Rate": 400,
+                            // "ENU-PCCursor.Punctuation": 3,
+                            // "ENU-PCCursor.Pitch": 11,
+                            // "ENU-JAWSCursor.Rate": 400,
+                            // "ENU-JAWSCursor.Punctuation": 3,
+                            // "ENU-JAWSCursor.Pitch": 11,
+                            // "ENU-MenuAndDialog.Rate": 400,
+                            // "ENU-MenuAndDialog.Punctuation": 3,
+                            // "ENU-MenuAndDialog.Pitch": 11
                         },
                         "options": {
-                            "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\2019\\Settings\\VoiceProfiles\\GPII.VPF"
+                            "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\2019\\Settings\\enu\\DEFAULT.JCF",
+                            "foldername": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\2019\\Settings\\VoiceProfiles"
```

Comment:
Given that this increases the number of places to replace the version, can we refactor these somehow?  Ideally there would be a `settingsDir` option and then sub-paths for the other settings.

Speaking of which, the naming here is not great.  I mean, these aren't "names" really, they're paths, and both are way too generic.  I'd suggest making the real meaning of each option plainer, like `mainSettingsFilePath` and `profileDirPath`.

Reply:
I agree that these names are awful, and I've renamed them to `defaultSettingsFilePath` and `voiceProfilesDirPath`. 

While the duplication of the version number isn't great, I think we should look at parameterize perhaps as part of the future SR work with grades, such that the preprocessing happened first and that the entire paths were still passed in to these options.  It seems unlikely that they would be different, but I don't necessary want to assume that the voice profiles directory would always be in the same area as the default voice profile.  Given the ever expanding set of JAWS options, and perhaps enterprise or roaming options, I'd like both of these final paths that go in to the settings handler to be the entire paths.

Reply:
I'm fine with that, but maybe leave a courtesy TODO next to the version strings?